client/src/components/Markdown/Elements/HistoryDatasetCollection/CollectionDisplay.vue

```diff
@@ -13,7 +13,7 @@
                     class="py-0 px-1">
                     <span class="fa fa-download" />
                 </b-button>
-                <UserHistories v-if="user" v-slot="{ currentHistoryId }" :user="currentUser">
+                <div v-if="currentUser">
```

Comment:
This divs condition can be attached to the button directly, if it is even still required

Reply:
I applied the change but I can't approve it while the collection display is not working #16048